### PR TITLE
Logo resize

### PR DESCRIPTION
**Pull request by susanthomson**
The logo and get involved message is now the size of a tweet in a 5 slot column. No longer overflows on fullscreen on my monitor. Added an option to have the banner logo in the bottom right.

### CODE REVIEW

**Review by stephen-lt-sl — APPROVED** [review]

File: client/angular/controllers/MainController.js [hunk 1]
```diff
@@ -161,22 +166,61 @@
...
+                "margin": tweetMargin + "px",
+                "width": logoBoxWidth + "px",
+                "height": logoBoxHeight + "px"
+            };
+        };
+
+        $scope.getLogoDimensions = function() {
+            return {
+                "width": logoBoxHeight + "px",
+                "height": logoBoxHeight + "px",
```
Comment (BRHoward) [inline client/angular/controllers/MainController.js:210]: width being set to height here

**Review by BRHoward — APPROVED** [review]
:last_quarter_moon_with_face:

File: client/angular/controllers/MainController.js (hunk 1 above)
Comment (susanthomson) [reply client/angular/controllers/MainController.js:210]: It's ensuring a square so the logo is nice and round :black_circle:

Comment (BRHoward) [reply client/angular/controllers/MainController.js:210]: oh right cool

**Review by BRHoward — APPROVED** [review]

**Pull request merged**